src/Webform.js

```diff
@@ -1084,7 +1084,7 @@ export default class Webform extends NestedDataComponent {
     }
     if (message) {
       this.alert = this.ce('div', {
-        class: classes || `alert alert-${type}`,
+        class: (options && options.classes) || `alert alert-${type}`,
```

Comment:
This needs to be reverse compatible... maybe something like.

```js
class: (typeof options === 'string') ? options : ((options && options.classes) || `alert alert-${type}`)
```

Reply:
The thing is, 'classes' parameter was introduced to be used in VPAT and recently I changed it to options to accommodate more customizable stuff into one object. I'm not sure if it needs to be reverse compatible, only if there is a scenario when an older version of VPAT is used with newest renderer.